tests/caesarlib.rs

```diff
@@ -55,8 +55,8 @@ mod tests {
     #[test]
     fn custom_base_sequence_correct_encipherd_and_deciphered() {
         let base = "zyxwvutsrqponmlkjihgfedcbaZYXWVUTSRQPONMLKJIHGFEDCBA";
-        let offset = 5;
         let test_word = "FooBar";
+        let offset = 5;
```

Comment:
Yes I guess it is good for the release.
I put them in the argument order (base, offset, test_word) which is also the alphabetical order, but it doesn't really matter I guess.